docs/integrations/storage/irys.md

```diff
@@ -60,61 +60,113 @@ dotenv.config();
-
 :::
 
-### Wallet signature
+### Connecting to Lit nodes
 
-A wallet signature (`authSig`) demonstrates true ownership of a wallet. By signing a basic transaction, regardless of its contents, you verify access to the wallet.
+Write a helper function to connect to a Lit node:
 
-First, create a file called `.env` with a single value, and include your private key.
+```js
+async function getLitNodeClient() {
+  // Initialize LitNodeClient
+  const litNodeClient = new LitJsSdk.LitNodeClientNodeJs({
+    alertWhenUnauthorized: false,
+    litNetwork: "cayenne",
+  });
+  await litNodeClient.connect();
 
-```json
-PRIVATE_KEY=
+  return litNodeClient;
+}
 ```
 
-Then, create a helper function that creates a message and signs it using your private key.
+### Obtain a Session Sigs
 
-```js
-async function getAuthSig() {
-  // Initialize the signer
-  const wallet = new ethers.Wallet(process.env.PRIVATE_KEY);
-  const address = ethers.utils.getAddress(await wallet.getAddress());
-
-  // Craft the SIWE message
-  const domain = "localhost";
-  const origin = "https://localhost/login";
-  const statement =
-    "This is a test statement. You can put anything you want here.";
-  const siweMessage = new siwe.SiweMessage({
-    domain,
-    address: address,
-    statement,
-    uri: origin,
-    version: "1",
-    chainId: "1",
-  });
-  const messageToSign = siweMessage.prepareMessage();
-
-  // Sign the message and format the authSig
-  const signature = await wallet.signMessage(messageToSign);
-  const authSig = {
-    sig: signature,
-    derivedVia: "web3.eth.personal.sign",
-    signedMessage: messageToSign,
-    address: address,
-  };
+In order to interact with the nodes in the Lit Network, you will need to generate and present session signatures. The easiest way to do this is to generate a `SessionSigs`. 
+
+`SessionSigs` are produced by a ed25519 keypair that is generated randomly on the browser and stored in local storage. We need to obtain an `AuthSig` through an authentication method like Ethereum wallet in order to get a `SessionSigs` from Lit Nodes.
+
+The session keypair is used to sign all requests to the Lit Nodes, and the user's `AuthSig` is sent along with the request, attached as a "capability" to the session signature. Each node in the Lit Network receives a unique signature for each request, and can verify that the user owns the wallet address that signed the capability.
+
+
+
+:::note
+Be sure to use the latest block hash from the `litNodeClient` as the nonce. You can get it from the `litNodeClient.getLatestBlockhash()`. Without the block hash SessionSigs will not be validated.
+:::
 
-  return authSig;
+```jsx
+import {ethers} from "ethers";
+import {
+  LitAccessControlConditionResource,
+  LitAbility,
+  createSiweMessageWithRecaps,
+  generateAuthSig,
+} from "@lit-protocol/auth-helpers";
+
+async getSessionSignatures(){
+    // Connect to the wallet
+    const ethWallet = new ethers.Wallet(
+      "<your private key>"
+    );
```

Comment:
why are we creating the wallet in scope of the session generation? This is a bit strange as the wallet should be outside of this scope and instead and `authSig` should be signed in the `authNeededCallback` so perhaps we pass the wallet to it?

Reply:
I've opted to just remove the code example from this doc and link to the `Get Session Sig` page - This helps us de-dup the code